Gemfile.lock

```diff
@@ -358,7 +358,7 @@ GEM
     parser (3.2.1.1)
       ast (~> 2.4.1)
     path_expander (1.1.1)
-    pg (1.4.5)
+    pg (1.5.0)
```

Comment:
Actually, this change is not required since Rails CI will remove the `Gemfile.lock` and perform a `bundle install`, but for clarity, Let's change the pg gem version to 1.5.1 that includes the fix for https://github.com/rails/rails/issues/48049
https://github.com/rails/buildkite-config/blob/cde08283a7f570c1b0185eb626f835e393350801/Dockerfile#L134